api/src/main/java/marquez/api/OpenLineageResource.java

```diff
@@ -130,6 +131,23 @@ public Response getLineageEvents(
     return Response.ok(new Events(events, totalCount)).build();
   }
 
+  @Timed
+  @ResponseMetered
+  @ExceptionMetered
+  @GET
+  @Consumes(APPLICATION_JSON)
+  @Produces(APPLICATION_JSON)
+  @Path("/runlineage/upstream")
+  public Response getRunLineageUpstream(
+      @QueryParam("runId") @NotNull RunId runId,
+      @QueryParam("depth") @DefaultValue(DEFAULT_DEPTH) int depth,
+      @QueryParam("facets") String facets) {
```

Comment:
Why are we pulling this in? I don't see it being used in the service. Did you intend to do something in the TODO block you left in there?

Reply:
Yes, my idea is to be able to select what facets to return for each dataset_version job_version and run in the result.
I can remove it for now.

Reply:
I removed this parameter in this iteration